test/integration/acoustic_comms.cc

```diff
@@ -161,6 +161,10 @@ INSTANTIATE_TEST_SUITE_P(
     // Message packets will be dropped as they are sent too fast
     // compared to "collision_time_interval".
     AcousticCommsTestDefinition(
-      "acoustic_comms_packet_collision.sdf", "addr2", "addr1", 3, 1)
+      "acoustic_comms_packet_collision.sdf", "addr2", "addr1", 3, 1),
+    // Source power is decreased and noise bumped up to make the packets
+    // drop.
+    AcousticCommsTestDefinition(
+      "acoustic_comms_propagation.sdf", "addr2", "addr1", 3, 0)
```

Comment:
Given the non-determinism in the system there is a teeny-tiny chance this _could_ become flaky. One possibility is to set  the [RNG to a fixed seed](https://github.com/gazebosim/gz-math/blob/a2f014e292a3b68e61384eb4bbea28ff4ad11553/include/gz/math/Rand.hh#L53), alternatively you could just check for a less than equal.

For the seed option exposing the seed to the end user may also help downstream users test individual cases better.